eureka_ml_insights/models/models.py

```diff
@@ -660,6 +660,9 @@ def pick_available_device(self):
 
             device = f"cuda:{gpu_index}"
 
+            if self.multi_gpu:
```

Comment:
multi-gpu should be added to HuggingFaceModel attributes, right? Otherwise all huggingface models break because they don't have this arg.